<?xml version='1.1' encoding='UTF-8'?>
<build>
  <actions>
    <hudson.model.CauseAction>
      <causeBag class="linked-hash-map">
        <entry>
          <hudson.model.Cause_-UserIdCause>
            <userId>admin</userId>
          </hudson.model.Cause_-UserIdCause>
          <int>1</int>
        </entry>
      </causeBag>
    </hudson.model.CauseAction>
    <hudson.plugins.sitemonitor.SiteMonitorRootAction plugin="sitemonitor@0.6">
      <mResults>
        <hudson.plugins.sitemonitor.model.Result>
          <mSite>
            <mUrl>http://localhost:8081/IgalNisanDanielNissim/</mUrl>
            <successResponseCodes/>
            <admitInsecureSslCerts>false</admitInsecureSslCerts>
          </mSite>
          <mStatus>EXCEPTION</mStatus>
          <mNote>[EXCEPTION] java.net.ConnectException: Connection refused (Connection refused) - Connection refused (Connection refused)</mNote>
        </hudson.plugins.sitemonitor.model.Result>
      </mResults>
    </hudson.plugins.sitemonitor.SiteMonitorRootAction>
  </actions>
  <queueId>19</queueId>
  <timestamp>1673804969048</timestamp>
  <startTime>1673804969111</startTime>
  <result>FAILURE</result>
  <duration>82</duration>
  <charset>UTF-8</charset>
  <keepLog>false</keepLog>
  <builtOn></builtOn>
  <workspace>/var/jenkins_home/workspace/monitoring</workspace>
  <hudsonVersion>2.375</hudsonVersion>
  <scm class="hudson.scm.NullChangeLogParser"/>
  <culprits class="java.util.Collections$UnmodifiableSet">
    <c class="sorted-set"/>
  </culprits>
</build>